src/Components/WebAssembly/JSInterop/src/Microsoft.JSInterop.WebAssembly.csproj

```diff
@@ -6,7 +6,7 @@
     <PackageTags>wasm;javascript;interop</PackageTags>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <IsPackable>true</IsPackable>
-    <IsShipping>true</IsShipping>
+    <IsShipping>false</IsShipping>
```

Comment:
@dougbu is this the same as IsShippingPackage ?

Reply:
Sort of. The Arcade turns one into the other at https://github.com/dotnet/arcade/blob/e7ede87875f41a9b3df898ae08da5ebc96e24f56/src/Microsoft.DotNet.Arcade.Sdk/tools/Version.BeforeCommonTargets.targets#L22